### ISSUE TITLE
Add text to TUTORIALS page

### ISSUE THREAD

**Issue by Gschwend** [linea-it/pz-server#70]
## Download 

To download a data product available on the Photo-z Server, go to one of the two pages by clicking on the card "**LSST PZ Data Products**" (for official products released by LSST DM Team) or "**User-generated Data Products**" (for products uploaded by the members of LSST community.  The download button is on the left side of each data product (each row of the list). 

## Upload

To upload a data product, click on the button "**NEW PRODUCT**" on the top left of the r "**User-generated Data Products**" page and fill in the Upload Form with relevant metadata. 

The photo-z-related products are organized into five categories (product types): 

* **Spec-z Catalog**: Catalog of spectroscopic redshifts and positions (usually equatorial coordinates). 
* **Training Set**: Training set for photo-z algorithms (tabular data). It usually contains magnitudes, errors, and true redshifts.
*  **Validation Set**: Validation (also known as test set) for photo-z algorithms (tabular data). It usually contains magnitudes, errors, and true redshifts. 
* **Photo-z Validation Results**: Results of a photo-z validation procedure (free format). Usually contains photo-z estimates (single estimates and/or pdf) of a validation set, photo-z validation metrics, validation plots, etc.
* **Photo-z Table**: Results of a photo-z estimation procedure. Ideally in the same format as the photo-z tables delivered by the DM as part of the LSST data releases. If the data is larger than the file upload limit (200MB), the product entry stores only the metadata (and instructions on accessing the data should be provided in the description field).

## Access via API 

The registered data products can also be accessed directly from Python code using the PZ Server's data access API. The internal documentation of the API functions is available on the [API's documentation page](https://linea-it.github.io/pz-server-lib/html/index.html).  

#### Instalation via pip

The PZ Server API is avalialble on **pip** as  `pz-server-lib`. To install the API and its dependencies, type, on the Terminal:  

```shell
$ pip install pz-server-lib 
``` 

#### Repository clone

Alternatively, if you have cloned the repository with:

```shell
$ git clone https://github.com/linea-it/pz-server-lib.git  
``` 

To install the API and its dependencies, type:

```shell
$ python setup.py install
```

## Tutorial notebook 

A Jupyter Notebook with instructions and use-case examples is available also in the [library's repository on Github](https://github.com/linea-it/pz-server-lib/blob/main/pz_server_tutorial.ipynb).

**Comment by Gschwend** [comment]
@crisingulani atualizei o texto, tinha um erro no comando de instalar com o setup.py
